tests/buffer_test.py

```diff
@@ -269,3 +272,245 @@ def teardown_class(cls):
+            assert "Buffer priority group 5 overlaps with existing priority group 5" in result.output
+
+        # Try to add an overlap entry
+        with mock.patch('utilities_common.cli.run_command') as mock_run_command:
+            result = runner.invoke(config.config.commands["interface"].commands["buffer"].commands["priority-group"].
+                                   commands["lossless"].commands["add"],
+                                   ["Ethernet0", "2-3"], obj=db)
+            print(result.exit_code, result.output)
+            assert result.exit_code
+            assert "Buffer priority group 2-3 overlaps with existing priority group 3-4" in result.output
+
+        # Try to add an overlap entry
+        with mock.patch('utilities_common.cli.run_command') as mock_run_command:
+            result = runner.invoke(config.config.commands["interface"].commands["buffer"].commands["priority-group"].
+                                   commands["lossless"].commands["add"],
+                                   ["Ethernet0", "4-5"], obj=db)
+            print(result.exit_code, result.output)
+            assert result.exit_code
+            assert "Buffer priority group 4-5 overlaps with existing priority group 3-4" in result.output
+
+        # Try to add a lossy profile
+        with mock.patch('utilities_common.cli.run_command') as mock_run_command:
+            result = runner.invoke(config.config.commands["interface"].commands["buffer"].commands["priority-group"].
+                                   commands["lossless"].commands["add"],
+                                   ["Ethernet0", "6", "ingress_lossy_profile"], obj=db)
+            print(result.exit_code, result.output)
+            assert result.exit_code
+            assert "Profile ingress_lossy_profile doesn't exist or isn't a lossless profile" in result.output
+
+        # Try to add a large pg
+        with mock.patch('utilities_common.cli.run_command') as mock_run_command:
+            result = runner.invoke(config.config.commands["interface"].commands["buffer"].commands["priority-group"].
+                                   commands["lossless"].commands["add"],
+                                   ["Ethernet0", "8", "ingress_lossy_profile"], obj=db)
+            print(result.exit_code, result.output)
+            assert result.exit_code
+            assert "Buffer priority group 8 is not valid" in result.output
+
+        # Try to use a non-exist profile
+        with mock.patch('utilities_common.cli.run_command') as mock_run_command:
+            result = runner.invoke(config.config.commands["interface"].commands["buffer"].commands["priority-group"].
+                                   commands["lossless"].commands["add"],
+                                   ["Ethernet0", "7", "testprofile"], obj=db)
+            print(result.exit_code, result.output)
+            assert result.exit_code
+            assert "Profile testprofile doesn't exist" in result.output
+
```

Comment:
One more test case - object map is not a number?

Reply:
`# Try to use a pg map in wrong format`
`# Try to use a pg which is not a number`